perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -585,6 +586,30 @@ def DisallowAllPorts(self):
     """Closes all ports on the firewall."""
     for firewall_rule in six.itervalues(self.firewall_rules):
       firewall_rule.Delete()
+    for firewall_rule in six.itervalues(self.firewall_icmp_rules):
+      firewall_rule.Delete()
+
+  def AllowIcmp(self, vm):
+    """Opens the ICMP protocol on the firewall.
+
+    Args:
+      vm: The BaseVirtualMachine object to open the ICMP protocol for.
+    """
+    if vm.is_static:
+      return
+    with self._lock:
+      firewall_name = ('perfkit-firewall-icmp-%s-%d-%d' %
+                       (FLAGS.run_uri, -1, -1))
+      key = (vm.project, -1, -1)
```

Comment:
There's no specifying of ports/types/codes here?  Can we just remove the -1 from the key and the name?

Reply:
Yes. The icmp firewall stuff in the gcloud cli doesn't have the same quirks as the aws one, so I've removed the ports from the key. 

I also added the vm cidr to the key, similar to the AllowPort() method so it doesn't have any collisions when there are multiple networks